lib/membrane_h264_plugin/parser.ex

```diff
@@ -109,18 +136,42 @@ defmodule Membrane.H264.Parser do
                 Repeat all parameter sets (`sps` and `pps`) on each IDR picture.
 
                 Parameter sets may be retrieved from:
-                  * The bytestream
+                  * The stream
                   * `Parser` options.
-                  * Decoder Configuration Record, sent as decoder_configuration_record
-                  in `Membrane.H264.RemoteStream` stream format
+                  * Decoder Configuration Record, sent in `:acv1` and `:avc3` stream types
+                """
+              ],
+              output_stream_structure: [
+                spec:
+                  nil
+                  | :annexb
+                  | :avc1
+                  | :avc3
+                  | {:avc1 | :avc3, nalu_length_size :: pos_integer()},
+                default: nil,
+                description: """
+                format of the outgoing H264 stream, if set to `:annexb` NALUs will be separated by
+                a start code (0x(00)000001) or if set to `:avc3` or `:avc1` they will be prefixed by their size.
```

Comment:
not necessarily, the `stable_prefixing?` option in NALuParser makes it reprefix with initial prefix. I like this behaviour because it is much easier to test (just compare buffers)